var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -33,6 +33,9 @@ class Flex(AutotoolsPackage):
     url = "https://github.com/westes/flex/releases/download/v2.6.1/flex-2.6.1.tar.gz"
 
     version('2.6.4', '2882e3179748cc9f9c23ec593d6adc8d')
+    # Build issue for v2.6.4 when gcc 7.2.0 is used.
+    # See issue #219; https://github.com/westes/flex/issues/219
+    conflicts('%gcc@7.2.0:', when='@2.6.4')
```

Comment:
could you please move this block below, maybe after all `depends_on` to be consistent with other packages?